### PR TITLE
Add posts and jobs feeds

### PR DESCRIPTION
**Pull request by gracelungu**
#### What does this PR do?
This PR adds the feeds for jobs and posts combined

#### Description of Task to be completed?
~ Add the feeds route and controller
~ Fetch combine and sort both posts and jobs by date

#### How should this be manually tested?
~ Fetch the branch and start the server
~ In the browser navigate to `localhost:3000/api/v1/feed`

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

**Comment by gracelungu** [comment]
[Get the landing page feeds](https://trello.com/c/Mh21Ij5o/32-get-the-landing-page-feeds)

**Review by oesukam — COMMENTED** [review]
@gracelungu Can't you also return feed related to organizations to be display on the side?

File: src/controllers/FeedsController.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  static async getFeeds(req, res) {
+    const jobs = await Job.find({})
+      .select('-__v')
+      .populate('author', '-_id -__v -userType -password');
+
+    const posts = await Post.find({})
+      .select('-__v')
+      .populate('author', '-_id -__v -userType -password');
+
+    const feeds = [...jobs, ...posts].sort(
```
Comment (oesukam) [inline src/controllers/FeedsController.js]: @gracelungu What about pagination?

**Comment by gracelungu** [comment]
@oesukam Do you mean adding an endpoint to get Organization feeds?

File: src/__tests__/routes/feeds.test.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import app from '../../app';
+import { urlPrefix } from '../../__mocks__/variables';
+import * as statusCodes from '../../constants/statusCodes';
+
+describe('Feeds', () => {
+  describe('Home Feeds', () => {
+    test('Should get Feeds', async () => {
+      const res = await request(app).get(`${urlPrefix}/feeds`);
+      expect(res.status).toBe(statusCodes.OK);
+      expect(res.body.feeds).toEqual(expect.any(Array));
```
Comment (oesukam) [inline src/__tests__/routes/feeds.test.js]: I think feed doesn't take an `s` in plural

**Review by oesukam — CHANGES_REQUESTED** [review]

**Comment by oesukam** [comment]
@gracelungu Is Travis still failing?

**Pull request merged**